README.md

```diff
@@ -44,13 +73,13 @@ For any other issues and feedback about this plugin, please submit it to this re
 
 ## Troubleshooting
 
-If you have trouble using plugins, check out the [Plugins Troubleshooting](https://docs.fastlane.tools/plugins/plugins-troubleshooting/) guide.
+If you have trouble using plugins, check out the [Plugins Troubleshooting guide](https://docs.fastlane.tools/plugins/plugins-troubleshooting/).
 
-## Using _fastlane_ Plugins
+## Using _Fastlane_ Plugins
 
-For more information about how the `fastlane` plugin system works, check out the [Plugins documentation](https://docs.fastlane.tools/plugins/create-plugin/).
+For more information about how the `Fastlane` plugin system works, check out the [Plugins documentation](https://docs.fastlane.tools/plugins/create-plugin/).
 
-## About _fastlane_
+## About _Fastlane_
 
-_fastlane_ is the easiest way to automate beta deployments and releases for your iOS and Android apps. To learn more, check out [fastlane.tools](https://fastlane.tools).
+_Fastlane_ is the easiest way to automate beta deployments and releases for your iOS and Android apps. To learn more, check out [Fastlane.tools](https://fastlane.tools).
```

Comment:
No need to change anything below ## Troubleshooting, as it was all auto generated by fastlane gem., this is the standard format for their gems.